.github/workflows/dependency-release.yaml

```diff
@@ -0,0 +1,47 @@
+name: CI
+
+on: repository_dispatch
+
+jobs:
+  bump_openstudio_gem:
+    name: "Check Event"
+    runs-on: ubuntu-latest
+    if: github.event.action == 'honeybee_openstudio_gem_release'
+
+    steps:
+    - name: "Checkout Master Branch"
+      uses: actions/checkout@v2
+      with:
+        ref: refs/heads/master
+        token: ${{ secrets.GH_TOKEN }}
+
+    - name: "Get Old Version"
+      run: |
+        export OLD_VERSION=$(gawk 'match($0, /honeybee-openstudio-gem==(.*)/, ary) {print ary[1]}'  ruby-requirements.txt)
+        echo "::set-env name=OLD_VERSION::$OLD_VERSION"
+      
+    - name: "Run Update Script"
+      env:
+        VERSION: ${{ github.event.client_payload.version }}
+      run: |
+        export CLEAN_VERSION=$(echo $VERSION | sed 's/v//g')
+        sed -i --regexp-extended 's/(honeybee-openstudio-gem==).*/honeybee-openstudio-gem=='"$CLEAN_VERSION"'/' ruby-requirements.txt
+
+    - name: "Set Commit Message"
+      id: vars
+      env:
+        VERSION: ${{ github.event.client_payload.version }}
+      run: |
+        echo ::set-output name=commit_message::"fix(deps): bump honeybee-openstudio-gem from v$OLD_VERSION to $VERSION"
```

Comment:
Can we change this to `chore(deps):`? We don't need a new release of this repo with each and every update to the honeybee-openstudio-gem. But we do need this version of the honeybee-openstudio-gem to be the latest whenever we eventually do a release.

Reply:
I'm about to head to bed but am happy with this change. Would you mind making the change and merging it in for me? :smile: